newFish.php

```diff
@@ -10,7 +10,7 @@
     && isset($_POST['color'])
     && isset($_POST['pattern'])
 ) {
-    insertEntryIntoDB($db,$_POST);
+   echo inputConfirmation(insertEntryIntoDB($db,$_POST));
```

Comment:
Consider loading this string into a variable and then echoing the variable in your pages' HTML body. The echo is quite difficult to do anything with styling-wise.